packages/split-ssh/src/split-ssh-formula/state/client/files/bashrc.d/sock.jinja

```diff
@@ -2,7 +2,7 @@
 # Ensure the SSH socket discoverability
 #
 # See /srv/salt/split-ssh in dom0 for details.
-SSH_VAULT_VM="ssh-vault"
+SSH_VAULT_VM="{{ pillar.split-ssh-vaults|first }}"
```

Comment:
This is a current **limitation**: even if multiple vaults were defined in the configuration (`/srv/user_pillar/config.yaml`), all the clients would be configured to use the **first** vault in the list.

_I didn't find any convenient way to specify which client should use which vault, and I am OK with this limitation until someone actually needs to use multiple vaults._

Reply:
**For future reference**: the correct syntax is (`pillar.get('split-ssh-vaults')` and the value must be made available in the clients for it to be accessible (in the top file: `split-ssh/init.jinja`). See https://github.com/gonzalo-bulnes/qubes-mgmt-salt-user/pull/10/commits/f3ea644f806dce3b0e6542fdf7b503689e735b6c